src/coreclr/jit/codegenxarch.cpp

```diff
@@ -5615,6 +5615,8 @@ void CodeGen::genCodeForStoreInd(GenTreeStoreInd* tree)
                         case NI_AVX512F_ExtractVector256:
                         case NI_AVX512DQ_ExtractVector128:
                         case NI_AVX512DQ_ExtractVector256:
+                        case NI_AVX10v1_V512_ExtractVector128:
```

Comment:
There's a few places in this file where we're asserting `assert(compiler->compIsaSupportedDebugOnly(InstructionSet_AVX512F))` that look like they should instead be checking for evex support or checking for `Avx512 || Avx10v1` support

Reply:
I see that now. true. I will update those checks accordingly. thanks for pointing that out. They never showed up in my testing because I am not testing a case where ```Avx512F``` is disabled and ```Avx10v1``` is enabled. I am yet to run testing with disabling ```Avx512F``` and ```PreferredVectorByteLength=32```. I will change this check and post updates testing as well when done.